importers/data_sources/importer.py

```diff
@@ -0,0 +1,192 @@
+"""
+Loads data sources into RE.
+This is a simple importer, as the data source load data is quite simple, 
+and also in it's ultimate form, requiring no transformation.
```

Comment:
typo: `s/it's/its/`

Reply:
thanks, yes, my laptop keyboard is falling apart and doubles up spaces all the time when it's in a bad mood.